source/_patterns/03-organisms/header/_header.scss

```diff
@@ -0,0 +1,12 @@
+// All component Sass needs non-printing base config
+@import '../../00-base/config';
+
+// To change default bootstrap jumbotron variables, SEE ~bootstrap/scss/variables
+// Add bootstrap variables here
+
+//$jumbotron-padding: some_override_value;
+//$jumbotron-bg: some_override_value;
+
+@import '~bootstrap/scss/jumbotron';
```

Comment:
If `jumbotron` already exists as a pattern, do **not** included it here. Just having the dependency in your `header/index.js` pulls over jumbotron styles. So, remove this line and just have your custom header sass if needed.

Reply:
Ah! My bad, we had a bit chat about this early on and time's made that a bit muddy. Removing ASAP.